core/trino-main/src/main/java/io/trino/sql/planner/optimizations/ReplicateJoinAndSemiJoinInDelete.java

```diff
@@ -13,20 +13,25 @@
  */
 package io.trino.sql.planner.optimizations;
 
+import com.google.common.collect.ImmutableList;
 import io.trino.Session;
 import io.trino.execution.warnings.WarningCollector;
 import io.trino.sql.planner.PlanNodeIdAllocator;
 import io.trino.sql.planner.SymbolAllocator;
 import io.trino.sql.planner.TypeProvider;
 import io.trino.sql.planner.plan.DeleteNode;
+import io.trino.sql.planner.plan.JoinNode;
+import io.trino.sql.planner.plan.JoinNode.Type;
 import io.trino.sql.planner.plan.PlanNode;
 import io.trino.sql.planner.plan.SemiJoinNode;
 import io.trino.sql.planner.plan.SimplePlanRewriter;
 
-import static io.trino.sql.planner.plan.SemiJoinNode.DistributionType.REPLICATED;
+import static io.trino.sql.planner.plan.JoinNode.DistributionType.REPLICATED;
```

Comment:
Don't import, just like you removed import for SemiJoinNode.DistributionType.REPLICATED